api-docs/swagger-v1.1.yml

```diff
@@ -1108,10 +1108,14 @@ parameters:
     required: false
   personal_health_scores:
     name: personal_health_scores[]
-    type: string
+    type: array
+    items:
+      type: string
+      enum: ['A', 'B1', 'B2', 'C1', 'C2']
     in: query
     description: restrict by health score, repeat parameter
     required: false
+    collectionFormat: multi
```

Comment:
Interesting, but that's it. I think we can omit the _, repeat parameter_ when using this :)